playbooks/common/openshift-node/config.yml

```diff
@@ -1,6 +1,8 @@
 ---
 - name: Gather and set facts for node hosts
-  hosts: oo_nodes_to_config
+  hosts:
+    - oo_nodes_to_config
+    - oo_infra_to_config
```

Comment:
Instead of defining these separately, should we just use a tag/inventory var to determine infra/node, etc?

Speaking of which, what base labels should we care about?  My changes for the templating do Region and Zone, we could add another category to differentiate Infra from non-Infra nodes and update the scheduler config accordingly.